packages/app/client/src/components/projects/overview/components/Team.js

```diff
@@ -0,0 +1,217 @@
+                  alt="Avatar"
+                  src={props.contactPerson.avatar}
+                />
+                <div>
+                  <Typography
+                    color="secondary"
+                    className={classes.personFullName}
+                    variant="subtitle1"
+                  >
+                    {props.contactPerson.name /* then + surname*/}
+                  </Typography>
+                  <Typography color="secondary" variant="body1">
+                    {props.contactPerson.email}
+                  </Typography>
+                </div>
+              </div>
+            </div>
+          ) : null}
+        </div>
+
+        <div>
+          <div className={classes.top}>
+            <Typography
+              color="secondary"
+              className={classes.teamSubtitle}
+              variant="subtitle1"
+            >
+              Team members
+            </Typography>
+            <Button
+              onClick={handleOpen}
+              className={classes.memberButton}
+              color="primary"
+            >
+              + Add members
+            </Button>
+            <TeamMembersModal
+              projectId={props._id}
+              team={props.team}
+              open={open}
+              setOpen={setOpen}
+              onClose={handleClose}
+            />
+          </div>
+          {props.team.map(member => (
+            <div key={Math.random()} className={classes.personInfo}>
+              <div style={{display: 'flex'}}>
+                <img
+                  className={classes.avatar}
+                  alt="Avatar"
+                  src={member.avatar}
+                />
+                <div>
+                  <Typography
+                    color="secondary"
+                    className={classes.personFullName}
+                    variant="subtitle1"
+                  >
+                    {member.name /* then + surname*/}
+                  </Typography>
+                  <Typography color="secondary" variant="body1">
+                    {member.email}
+                  </Typography>
+                </div>
+              </div>
+              <Button onClick={() => deleteHandler(member._id)} color="secondary" className={classes.deleteButton}><img alt="delete icon" src={DeleteButton}/></Button>
+            </div>
+          ))}
+        </div>
+        <div>
+          <Button color="primary" onClick={clickHandler}>
+            Save
+          </Button>
+          <Button onClick={cancelHandler} color="secondary" variant="outlined">
+            Cancel
+          </Button>
+        </div>
+      </div>
+    </Container>
+  )
+}
+
+Team.propTypes = {
+  refreshToken: PropTypes.func.isRequired,
+  deleteMemberFromProject: PropTypes.func.isRequired
+}
+
+const mapStateToProps = state => ({
+  auth: state.auth
+})
+
+export default compose(connect(mapStateToProps, { refreshToken, deleteMemberFromProject }))(Team)
```

Comment:
Why are we using these HOC? We could use the React hooks. If we still want to use the connect the compose HOC in this particular case is not needed because we just have the connect HOC being used there is no other to compose.

Reply:
Because I made it by example, but I'm agree that its better to use React hooks